build.sbt

```diff
@@ -87,7 +87,7 @@ libraryDependencies ++= Seq(
     "com.gu" %% "content-api-models" % capiModelsVersion,
     "com.gu" %% "content-api-models-json" % capiModelsVersion,
     "com.gu" %% "content-api-client-aws" % "0.5",
-    "com.gu" %% "editorial-permissions-client" % "0.8",
+    "com.gu" %% "editorial-permissions-client" % "2.0-SNAPSHOT",
```

Comment:
Should this be the `SNAPSHOT` version?

Reply:
Yeah this was for local testing, I'll update it once I've published